lib/fatfs/ffconf.h

```diff
@@ -97,7 +97,7 @@
 */
 
 
-#define FF_USE_LFN		0
+#define FF_USE_LFN		1
```

Comment:
Was this really required?
It's obviously something that might impact on every other board out there, so ideally if this could not be changed would be preferable.